tools/gptfdisk/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gptfdisk
```

Comment:
I mean this change and related changes, which introduces ``gdisk`` should be submitted as separate pull request.